### PR TITLE
fix(ci): update README deps and build.yml

### PR DESCRIPTION
**Pull request by gerblesh**
This PR updates the CI, removes COPR build and updates the docs to include topgrade as a dependency.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -13,20 +13,20 @@ env:
 jobs:
   push-ghcr:
     name: Build and push image
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```
Comment (m2Giles) [inline .github/workflows/build.yml:16]: change this to 24.04 to fix the tar bug.

Comment (gerblesh) [reply .github/workflows/build.yml:16]: thanks

File: Containerfile.builder [hunk 2]
```diff
@@ -17,7 +19,7 @@ RUN dnf install \
     rpm-build && \
     mkdir -p "$UBLUE_ROOT" && \
     rpkg spec --outdir  "$UBLUE_ROOT" && \
-    dnf builddep -y output/ublue-update.spec
+    dnf builddep -y output/ublue-update.spec \
```
Comment (m2Giles) [inline Containerfile.builder:22]: This \ needed?

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -139,21 +139,3 @@ jobs:
         if: github.event_name != 'pull_request'
         run: |
           echo "${{ toJSON(steps.push.outputs) }}"
-
-  copr-build:
```
Comment (m2Giles) [inline .github/workflows/build.yml:143]: Don't know the history, but why is this getting removed?

Comment (gerblesh) [reply .github/workflows/build.yml:143]: it was originally supposed to be a github action that would automatically tell a COPR to rebuild, it was abandoned a while ago and now the COPR automatically rebuilds on new releases

File: Containerfile.builder (hunk 2 above)
Comment (gerblesh) [reply Containerfile.builder]: not needed, removed. Thanks!

**Review by m2Giles — APPROVED** [review]

**Pull request merged**